hfo2/src/panic.rs

```diff
@@ -25,7 +25,10 @@ fn abort_impl() -> ! {
 /// system in an inconsistent state.
 ///
 /// TODO: Should this also reset the system?
+/// TODO(HfO2): This function is marked `linkage=weak`, because some tests have custom `abort`
+/// function but still need HfO2. Separating HfO2 into many libraries may resolve this.
```

Comment:
- "This function needs to be weakly linked because ..."
- "Dividing HfO2 into ..."